cog/daily_charge.py

```
@@ -23,95 +26,242 @@
 
 
 class Charge(commands.Cog):
-    def __init__(self, bot):
+    def __init__(self, bot: discord.Bot):
         self.bot = bot
-        self.embed = None
 
-    async def send_message(self, point, combo, interaction):
+    async def restore_downtime_point(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `restore_downtime_point`

Add return type annotation: `None`

(ANN201)